templates/dart/lib/src/chunked_upload_io.dart.twig

```diff
@@ -48,7 +64,15 @@ Future<Response> chunkedUpload({
     if(offset < size) {
       headers['x-appwrite-id'] = res.data['\$id'];
     }
-    onProgress?.call(min(offset,size)/size * 100);
+    final progress = UploadProgress(
+      $id: res.data['\$id'] ?? '',
```

Comment:
Why the `?? ''` part? We should have ID always, right?

Reply:
Because we don't have that for mock endpoint and tests fail.